addons/stock_account/models/product.py

```diff
@@ -851,4 +859,4 @@ def write(self, vals):
     @api.onchange('property_valuation')
     def onchange_property_valuation(self):
         # Remove or set the account stock properties if necessary
-        self._check_valuation_accouts()
+        self._check_valuation_accounts()
```

Comment:
An idea why is that an onchange instead of an extra field in the @api.constrains ?

Reply:
it's already the case, but without this onchange it still can't update the accounts